pkg/apis/admissionregistration/types.go

```diff
@@ -929,3 +949,26 @@ type ServiceReference struct {
 	// +optional
 	Port int32
 }
+
+// MatchCondition represents a condition which must by fulfilled for a request to be sent to a webhook.
+type MatchCondition struct {
+	// Name is an identifier for this match condition, used for strategic merging of MatchConditions,
+	// as well as providing an identifier for logging purposes. A good name should be descriptive of
+	// the associated expression.
+	// Name must be a valid RFC 1123 DNS subdomain, and unique in a set of MatchConditions.
+	// Required.
+	Name string
+
+	// Documentation on CEL: https://kubernetes.io/docs/reference/using-api/cel/
+	//
+	// Expression represents the expression which will be evaluated by CEL.
+	// ref: https://github.com/google/cel-spec
+	// CEL expressions have access to the contents of the AdmissionRequest, organized into CEL variables:
+	//
+	// 'object' - The object from the incoming request. The value is null for DELETE requests.
+	// 'oldObject' - The existing object. The value is null for CREATE requests.
+	// 'request' - Attributes of the admission request([ref](/pkg/apis/admission/types.go#AdmissionRequest)).
```

Comment:
Add authorizer to the list .

Reply:
added authorizer and authorizer.requestResource similar to VAP